src/components/GameView/PointCounter.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <div class="point-counter">
+    <div
+      class="point-counter__wrapper"
+      :class="[isPlayer ? 'point-counter__player' : 'point-counter__opponent']"
+    >
+      <div
+        class="d-flex flex-column align-center point-counter__inner-container"
+        :class="[isPlayer ? 'point-counter__player' : 'point-counter__opponent']"
+      >
+        <div
+          class="point-counter__current-points"
+          :class="[ isPlayer ? 'point-counter__current-player' : 'point-counter__current-opponent' ]"
+        >
+          {{ currentPoints }}
+        </div>
+        <div class="point-counter__total-points">{{ pointsToWin }}</div>
+        <v-menu :location="isPlayer ? 'top' : 'bottom'">
+          <template #activator="{ props }">
+            <v-btn class="point-counter__btn" icon v-bind="props" variant="plain">
+              <v-icon color="black" icon="mdi-information" v-bind="iconSize"/>
+            </v-btn>
+          </template>
+          <v-list class="score-goal-explanation">
+            <v-list-item :class="{ 'current-goal': kingCount === 0 }"> 0 Kings: 21pts </v-list-item>
+            <v-list-item :class="{ 'current-goal': kingCount === 1 }"> 1 King: 14pts </v-list-item>
+            <v-list-item :class="{ 'current-goal': kingCount === 2 }"> 2 Kings: 10pts </v-list-item>
+            <v-list-item :class="{ 'current-goal': kingCount === 3 }"> 3 Kings: 5pts </v-list-item>
+            <v-list-item :class="{ 'current-goal': kingCount === 4 }"> 4 Kings: 0pts </v-list-item>
+          </v-list>
```

Comment:
This should become a computed prop since this is a net new component. I'd like to see something like this as a computed prop:
```
kingPointsToWin() {
    return [0, 5, 10, 14, 21];
},
```

And then something like this in the template:
```suggestion
          <v-list class="score-goal-explanation">
            <v-list-item v-for="(points, idx) in kingPointsToWin" :class="{ 'current-goal': kingCount === idx }"> {{ idx }} Kings: {{ points }}pts </v-list-item>
          </v-list>
```